react/src/AddDatasets/index.tsx

```diff
@@ -344,7 +344,9 @@ export default function AddDatasets() {
             currCols,
             RNA_ONLY_FIELDS.map(field => ({
                 field,
-                hidden: OPTIONAL_RNA_FIELDS.includes(field) && field !== "vcf_available",
+                hidden:
+                    OPTIONAL_RNA_FIELDS.includes(field) &&
+                    !["vcf_available", "candidate_genes"].includes(field),
```

Comment:
I think we can actually just leave this off as `OPTIONAL_RNA_FIELDS = ["vcf_available", "candidate_genes"]`